itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestExportImport.java

```diff
@@ -106,6 +113,54 @@ public void dataImportAfterMetadataOnlyImport() throws Throwable {
         .verifyResults(new String[] { "1", "2" });
   }
 
+  @Test
+  public void testExportExternalTableSetFalse() throws Throwable {
+    String path = "hdfs:///tmp/" + dbName + "/";
+    String exportMDPath = "'" + path + "1/'";
+    String exportDataPath = "'" + path + "2/'";
+    String exportDataPathRepl = "'" + path + "3/'";
+    srcHiveWarehouse.run("create external table " + dbName + ".t1 (i int)")
+            .run("insert into table " + dbName + ".t1 values (1),(2)")
+            .run("export table " + dbName + ".t1 to " + exportMDPath + " for metadata replication('1')")
+            .run("export table " + dbName + ".t1 to " + exportDataPath)
+            .runFailure("export table " + dbName + ".t1 to " + exportDataPathRepl + " for replication('2')");
+
+    destHiveWarehouse.run ("use " + replDbName)
+            .run("import table " + replDbName + ".t1 from " + exportMDPath)
+            .run("show tables like 't1'")
+            .verifyResult("t1")
+            .run("import table " + replDbName + ".t2 from " + exportDataPath)
+            .run("select * from " + replDbName + ".t2")
+            .verifyResults(new String[] { "1", "2" })
+            .runFailure("import table " + replDbName + ".t3 from " + exportDataPathRepl)
```

Comment:
Export already failed for this path exportDataPathRepl. No point in importing from this path.

Reply:
its ok to have import